test/ingress/conformance/tests/httproute-ignore-case-match.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"testing"
+
+	"github.com/alibaba/higress/test/ingress/conformance/utils/http"
+	"github.com/alibaba/higress/test/ingress/conformance/utils/suite"
+)
+
+func init() {
+	HigressConformanceTests = append(HigressConformanceTests, HTTPRouteIgnoreCaseMatch)
+}
+
+var HTTPRouteIgnoreCaseMatch = suite.ConformanceTest{
```

Comment:
please add HTTPRouteIgnoreCaseMatch to [e2e_test.go](https://github.com/alibaba/higress/blob/main/test/ingress/e2e_test.go#L53)

Reply:
Because this could be referred as a separated pkg to use.